### ISSUE TITLE
Blank Page and stuff

### ISSUE THREAD

**Issue by pierrickvr** [tananaev/traccar#3649]
Hi,
First, thank you for Traccar !!!!!

I have a synology DS214+.
I install Traccar but when I run it, there are a blank page.
In the Admin consol, it's write this :
```
Uncaught Error: You're trying to decode an invalid JSON String: 
    at new Ext.Error (cdnjs.cloudflare.com/ajax/libs/extjs/6.0.1/ext-all.js:22)
    at Function.raise (cdnjs.cloudflare.com/ajax/libs/extjs/6.0.1/ext-all.js:22)
    at Object.Ext.raise (cdnjs.cloudflare.com/ajax/libs/extjs/6.0.1/ext-all.js:22)
...
```
I'm use the 55000 port.
I'm open it on my FAI Box and It's not blocked on my Synology Firewall.

I tested the Windows manager and it's run.

Can you help me ?
My mail : pierrickvr@gmail.com

Thanks you very much

**Comment by tananaev** [comment]
Closing as duplicate of the forum thread:

https://www.traccar.org/forums/topic/blank-page-and-stuffs/
